bokeh/models/map_plots.py

```diff
@@ -83,10 +83,6 @@ def _check_missing_renderers(self):
     def _check_no_glyph_renderers(self):
         pass
 
-    map_options = Instance(GMapOptions, help="""
-    Options for displaying the plot.
-    """)
-
```

Comment:
possibly we should remove the one in the base class instead? It's wrong to have it in both base and sub class with different types though, for sure.